doc/source/getting_started/install.rst

```diff
@@ -85,6 +85,10 @@ as well::
 
     pip install geopandas
 
+Optional runtime dependencies can also be installed at once::
```

Comment:
```suggestion
Optional runtime dependencies can also be installed all at once::
```

I'd also suggest putting this after the warning text just below this - as the warning is for required deps.

Reply:
Thanks, done!